test/boilerplate/test-index.js

```diff
@@ -1,26 +1,8 @@
 window.MockHttpServer = require('./mockhttprequest.js');
 window.Should = require('should');
-
-var mp = function () {
-    var self = this;
-
-    this.addForwarder = function (forwarder) {
-        self.forwarder = new forwarder.constructor();
-    };
-
-    this.getCurrentUser = function() {
-        return currentUser();
-    };
+require('../../node_modules/mparticle-sdk-javascript/src/main.js');
+mParticle.addForwarder = function (forwarder) {
+    mParticle.forwarder = new forwarder.constructor();
 };
-
-function currentUser() {
-    return {
-        getMPID: function() {
-            return 123;
-        }
-    };
-}
-
-window.mParticle = new mp();
-require('../../src/mParticleJSKitBoilerplate.js');
+require('../../node_modules/@mparticle/web-kit-wrapper/index.js');
```

Comment:
rather than stub mparticle. i am including the full sdk to have access to helper methods such as `mParticle.eCommerce.expandCommerceEvent`